src/lib.rs

```diff
@@ -14,4 +14,6 @@
 //! [simulator]: https://github.com/embedded-graphics/simulator
 //! [setup]: https://github.com/embedded-graphics/simulator#setup
 
+extern crate alloc;
```

Comment:
This in included with the compiler, but you need to do this to get access to it in a `no_std` environment.